scripts/jenkins/wait-for-cluster.sh

```diff
@@ -0,0 +1,4 @@
+until kubectl cluster-info
```

Comment:
missing she-bang and it's always good to have a quick description of what this script does ... generally following https://google.github.io/styleguide/shell.xml